### PR TITLE
Format code segments in the handbook #79

### LINKED ISSUE #79: Format code segments in the handbook
Currently, some code segments are shown as screenshots and some are formatted as normal text. They can be formatted better, using the same technique used in http://www.comp.nus.edu.sg/~cs2103/AY1516S1/contents/testingFaq.html

### PR DESCRIPTION
**Pull request by vhazali**
Fixes #79 
[Preview](http://vhazali.github.io/website)

### CODE REVIEW

**Comment by damithc** [comment]
What about the code screenshot in CE1? Any chance that can be converted to code as well?
Resist the temptation to fix multiple things in the same PR. If you fix multiple things in the same PR, it becomes harder to undo one thing while keeping the rest.

**Comment by vhazali** [comment]
Will work on the image in CE1!
Not quite sure about the multiple things, did you mean the commit on changing the grammar?

**Comment by damithc** [comment]
Yes I meant that one, and the changes to indentation in unrelated lines. If you notice other things that need fixing while fixing something else, just create an issue for it. More issues mean more opportunities for others to contribute :-)

**Comment by vhazali** [comment]
Noted! Sorry about that

**Comment by damithc** [comment]
Post a comment when ready for review again.

**Comment by vhazali** [comment]
Yup, ready!

File: contents/handbook-theProduct.html [hunk 1]
```diff
@@ -1,86 +1,86 @@
...
+     <ul>
+      <li>Jim is currently using Google calendar. Because he store/retrieve/edit todo tasks so frequently,
+       he find the online calendar too slow to work with. He’d rather have a desktop software he can summon
+       quickly. Even better (but not a must) if he can activate the software by pressing a keyboard shortcut.
+       He is quite impressed by how the <a href="http://www.launchy.net/">Launchy tool can be activated 
+       easily by pressing Alt+Space</a>.</li>
+       <li>A calendar is not very good in capturing tasks that need to be done before a specific date/time,
+        or after a specific date/time, and items without specific times.</li>
+        <li>Entering an event in a calendar is a pain because it takes several clicks.  He prefers a ‘one shot’
+        approach. e.g., to be able to type in commands such as "<kbd>add July 10, 5-6, project meeting</kbd>"
```
Comment (damithc) [inline contents/handbook-theProduct.html]: Why `<kbd>` not `<code>`?

**Comment by damithc** [comment]
Added one comment. @wkurniawan07 anything to add?

**Comment by vhazali** [comment]
Was trying out the difference between the two, forgot to change it back to ```<code>``` changed and recommitted

File: contents/handbook-ce2.html [hunk 2]
```diff
@@ -1,163 +1,156 @@
...
+   </li>
+   <li>
+     Having a public method that takes the user command and executes it (CityConnect uses this
+     approach) can greatly simplify the testing process. Then, you can write tests like this:
+   </li>
+   <pre class="prettyprint"><code class="lang-java">//check if the “clear” command returns the right status message
+    assertEquals(“all content deleted from mytextfile.txt”,
+    TextBuddy.executeCommand(“clear”));
+    //check if the file was actually cleared
+    assertEquals(0, TextBuddy.getLineCount());</code></pre>
```
Comment (wkurniawan07) [inline contents/handbook-ce2.html]: There shouldn't be any indentation here, because they'll be rendered as such (not ignored) in `pre`.

File: contents/handbook-v04Product.html [hunk 3]
```diff
@@ -1,77 +1,92 @@
...
+        Here is a sample code file in java:
+      </p>
+
+      <pre class="prettyprint"><code class="language-java">//@author A0909865T
+method 1 ...
+method 2 ...
+//@author A0702345U
+method 3 ...
+//@author A0909865X
+method 4 ...</code></pre>
```
Comment (wkurniawan07) [inline contents/handbook-v04Product.html]: The examples all used single `@` here.

File: contents/handbook-faq-noLaptop.html [hunk 4]
```diff
@@ -1,4 +1,4 @@
 <p>If you do not have a laptop or prefer not to bring the laptop, it is up to you to show your 
 work to the tutor in some way (e.g. by connecting to your home PC remotely), *without* requiring 
 extra time/effort from the tutor or team members.</p>
-<p>Reason: You enjoy the benefits of not bring the laptop; you should bear the cost too</p>
+<p>Reason: You enjoy the benefits of not bringing the laptop; you should bear the cost too</p>
```
Comment (wkurniawan07) [inline contents/handbook-faq-noLaptop.html]: This should not be part of this PR.

File: styles/handbook.css [hunk 5]
```diff
@@ -8,4 +8,23 @@ h2{
...
+
+pre.cliprint{
+	background-color: #000; /*black*/
+	-moz-border-radius: 8px;
+	-webkit-border-radius: 8px;
+	-o-border-radius: 8px;
+	-ms-border-radius: 8px;
+	-khtml-border-radius: 8px;
+	border-radius: 8px;
+	color:#ffec00;
```
Comment (wkurniawan07) [inline styles/handbook.css]: Add some padding also. Currently the text content sticks to the boundaries.

**Comment by wkurniawan07** [comment]
On top of what I've commented on, try your best to revert all the indentation changes made.

**Comment by vhazali** [comment]
Regarding the reverting of indentation changes, can I confirm that we don't want the 2 spaces as indentation? It is kind of contradicting what's said on the readme.md

**Comment by damithc** [comment]
No, 2 spaces is correct. The reason for reverting is because this PR is not about correcting indentation.

**Comment by vhazali** [comment]
Ahh! alright, I'll try my best to revert all of them

**Comment by vhazali** [comment]
Ready for next review!

File: contents/handbook-theProduct.html [hunk 6]
```diff
@@ -8,79 +8,79 @@
...
+						 </ul>
+					 </p>
+					 <p>
+
+						 Design your product to solve some of Jim’s problems to make his workflow even more efficient and painless. i.e.
+						 to make his life better. You don’t have to solve all problems mentioned above and there may be other problems you
+						 could solve but not mentioned above. Good software delights users by solving problems users don’t even realize they have.
+						 You can also draw inspiration from the <a href="http://todotxt.com/">Todo.txt project</a>, but try to do 
+						 better.
+					 </p>
```
Comment (damithc) [inline contents/handbook-theProduct.html]: This file contains lot of unrelated changes. Use this as an opportunity to learn how to clean up a PR/branch :-)

File: contents/handbook-ce1.html [hunk 7]
```diff
@@ -2,7 +2,50 @@
 	Write a CLI (Command Line Interface) program called TextBuddy using Java/C++ to manipulate text in a file.
 	Here is a sample interaction with the program. Text in purple are commands entered by the user.
 </p>
-<img src="../images/ce1_CLI.PNG">
+
+<pre class="cliprint">
+<code class="clisystem">c:></code> <code class="cliuser">TextBuddy.exe mytextfile.txt</code> (OR <code class="clisystem">c:>java</code> <code class="cliuser">TextBuddy mytextfile.txt</code>)
```
Comment (wkurniawan07) [inline contents/handbook-ce1.html]: I think `java` is also part of the user input, yea?

File: contents/handbook-v04Product.html [hunk 8]
```diff
@@ -18,60 +18,75 @@
...
-				that were trivially copy pasted e.g. a test data file with 1000 tasks that was used to test performance.
+				Download the <a href="https://github.com/collate/collate" target="_blank">collate tool</a>.
+			</li>
+			<li>
+				Mark your code with a <code>//@@author matric-number</code>. Note the double <code>@</code>. The comment syntax may vary
+				based on file type e.g. for html
+				<pre class="prettyprint"><code class="lang-html"><<!--\-->!-- @@author matric-number --></code></pre>
+			</li>
+			<p>
+				Here is a sample code file in java:
```
Comment (wkurniawan07) [inline contents/handbook-v04Product.html]: The prettyprint did use java, but I don't think this sample code is java-specific.

File: contents/handbook-v04Product.html [hunk 9]
```diff
@@ -18,60 +18,75 @@
...
+				</li>
+					e.g. <pre class="prettyprint"><code class="lang-java">//@@author A0909865T-reused
+method 1 ...
+method 2 ...</code></pre>
+					where the matric number belongs to the person responsible for reusing that code.
+				<li>
+					<strong> EXCLUDE</strong> code generated by the IDE/framework. You may also mark such code
+					<code>//@@author generated</code></li>
+					e.g.
+					<pre class="prettyprint"><code class="lang-java">//@@author generated
```
Comment (wkurniawan07) [inline contents/handbook-v04Product.html]: Similar for these two above.

**Comment by wkurniawan07** [comment]
Just left with the minor issues there, and unnecessary indentation changes. You can use [the files changes tab](https://github.com/nus-cs2103/website/pull/103/files) to see the changes you've made in this PR.

**Comment by vhazali** [comment]
I think there's no more indentation errors.
Ready for review

**Comment by wkurniawan07** [comment]
It's good now. Well done for this :+1:
